### DISCUSSION TITLE
Is there a way to not send a scope

### DISCUSSION THREAD

**Discussion by marmundo** [nextauthjs/next-auth#10680] [Help] [answered]
Hello,

I'm creating a provider. Auth.js uses scope in query to /authorize. However, scope is generating problem to me.

Is there a way to not send a scope as `https://domain/o/authorize?client_id=fz&response_type=code&grant_type=authorization-code`

If I define scope as '', it completes with openid: `https://domain/o/authorize?client_id=fz&response_type=code&grant_type=authorization-code?scope=openid`
```
  authorization: {
        url: `domain`,
        params: { scope: '' },
```

What is the problem sending scope?

I get a ` Text: {"error": "invalid_request", "error_description": "Code verifier required."}`error. Making request without scope I get access token.

Thanks.

**Comment by davidpc-pg** [comment, marked as answer]
That error looks related to PKCE... If you're not using PKCE, I think you need to set the `checks` property of your provider to something else (like `'none'`), because it defaults to `'pkce'`.
